virt_lightning/__init__.py

```diff
@@ -159,25 +159,31 @@ def start(self, domain):
         domain.attachDisk(meta_data_iso, device="cdrom", disk_type="raw")
         domain.dom.create()
 
+    def find_kvm_binary(self):
+        paths = ["/usr/bin/qemu-kvm", "/usr/bin/kvm"]
+        for i in paths:
+            if os.path.exists(i):
+                return
+        else:
+            raise Exception("Failed to find the kvm binary in: ", paths)
+
 
 class LibvirtDomain:
-    def __init__(self, dom, hv=None):
+    def __init__(self, dom):
         self.dom = dom
-        self.cloud_init = []
+        self.cloud_init = {
```

Comment:
Will code work with changes from list to dict? I didn't check all code but it looks like won't work.

Reply:
it was actually a mistake. cloud_init key is a `dict()`.